### PR TITLE
Movement refactor#248

### PR DESCRIPTION
**Pull request by AleksandrShvartz**

### CODE REVIEW

File: website/assets/js/blockRedactor.js [hunk 1]
```diff
@@ -337,61 +319,200 @@ export class BlockRedactor {
...
+                // Прозрачность обводки.
+                strokeOpacity: 0.8,
+                // Ширина обводки в пикселях.
+                strokeWidth: 3,
+            });
+            myMap.geoObjects.add(myCircle);
+            myMap.geoObjects.add(placeMark);
+        }
+    }
+    static addMethodsToMap(myMap, question, res, quest) {
```
Comment (BaevDaniil) [inline website/assets/js/blockRedactor.js:367]: Why is the quest passed here??

File: website/assets/js/blockRedactor.js [hunk 2]
```diff
@@ -337,61 +319,200 @@ export class BlockRedactor {
             modal.hide();
         };
     }
+    static addOldPlaces(myMap, quest, newQuestion) {
```
Comment (BaevDaniil) [inline website/assets/js/blockRedactor.js:322]: We agreed not to add such functionality. Why was it added?

Comment (BaevDaniil) [reply website/assets/js/blockRedactor.js:322]: Besides, it doesn't work.

**Comment by aVorotnikov** [comment]
@AleksandrShvartz don't forget to resolve merge conflicts manually

- [ ] Conflicts resolved

**Comment by BaevDaniil** [comment]
There are too many bugs and merge-related changes in this branch. Move to the new [branch](https://github.com/Quest-maker-team/quest-maker/tree/refactor_movement) and delete this

**Pull request closed without merging**